### PR TITLE
feat: input component page

### PR DESCRIPTION
**Pull request by hlavkat**

### CODE REVIEW

File: src/routes/components/input/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+	})
+</script>
+
+{#snippet description()}
+	Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem numquam aliquid fuga aperiam iure
+	quae neque a omnis delectus explicabo modi nam optio placeat eveniet dolores debitis accusantium
+	commodi asperiores voluptatem, adipisci alias ipsa inventore quos harum? Eaque eligendi, velit
+	atque porro accusantium, eveniet temporibus culpa exercitationem quas nobis iure?
+{/snippet}
+{#snippet examples()}{/snippet}
```
Comment (vojtechsimetka) [inline src/routes/components/input/+page.svelte:115]: Please add some examples.

File: src/routes/components/input/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+		css = await response.text()
+	})
+</script>
+
+{#snippet description()}
+	Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem numquam aliquid fuga aperiam iure
+	quae neque a omnis delectus explicabo modi nam optio placeat eveniet dolores debitis accusantium
+	commodi asperiores voluptatem, adipisci alias ipsa inventore quos harum? Eaque eligendi, velit
+	atque porro accusantium, eveniet temporibus culpa exercitationem quas nobis iure?
+{/snippet}
```
Comment (vojtechsimetka) [inline src/routes/components/input/+page.svelte:114]: Remove lorem ipsum

File: src/routes/components/input/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+{#snippet implement()}
+	<TabBar dimension="small">
+		<TabContent value="Svelte"><Code language="svelte" code={input} /></TabContent>
+		<TabContent value="CSS"><Code language="css" code={css} /></TabContent>
+	</TabBar>
+{/snippet}
+
+<ComponentTemplate
+	name="Input"
+	tagline="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum, quae?"
```
Comment (vojtechsimetka) [inline src/routes/components/input/+page.svelte:196]:
```suggestion
	tagline="
Input elements allow people to enter and edit text or data within a form."
```

**Review by vojtechsimetka — CHANGES_REQUESTED** [review]

**Review by vojtechsimetka — APPROVED** [review]

File: src/routes/components/input/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+	{#if unit}
+		<Input label="Add unit" bind:value={stringUnit} />
+	{/if}
+	<Switch bind:checked={errorMessage} label="With error" />
+	{#if errorMessage}
+		<Input label="Add error" bind:value={stringErrorMessage} />
+	{/if}
+	<Switch bind:checked={controlButton} label="With controls" />
+	{#if controlButton}
+		<Checkbox bind:checked={colorButton} label="Color palete button" />
```
Comment (agazso) [inline src/routes/components/input/+page.svelte:182]:
```suggestion
		<Checkbox bind:checked={colorButton} label="Color palette button" />
```

File: src/routes/components/input/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+		display: flex;
+		flex-direction: row;
+		justify-content: center;
+		align-items: center;
+		background-color: var(--colors-ultra-low);
+	}
+	:global(.preview-tab) {
+		height: 288px;
+	}
+	/* .example-row {
```
Comment (agazso) [inline src/routes/components/input/+page.svelte:257]: Please remove commented code.

File: src/routes/components/input/+page.svelte [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+		display: flex;
+		flex: 1;
+		flex-wrap: wrap;
+		align-items: center;
+		gap: 16px;
+		margin-top: 0px;
+		margin-bottom: 0px;
+	} */
```
Comment (agazso) [inline src/routes/components/input/+page.svelte:265]:
```suggestion
```

**Review by agazso — APPROVED** [review]
Mostly good, see my comments.

I think we should also add the Button component source code to the implementation, because the Input depends on that.

**Pull request merged**